include/lbann/utils/protobuf/impl.hpp

```diff
@@ -120,6 +121,27 @@ auto get_repeated_field_ref(google::protobuf::Message const& msg,
 } // namespace protobuf
 } // namespace lbann
 
+template <typename T, typename ContainerT>
+void lbann::protobuf::assign_to_repeated(google::protobuf::RepeatedField<T>& field,
+                        ContainerT const& values)
+{
+  /** @todo Change to Assign if older versions of protobuf are no
+   *  longer supported.
+   */
```

Comment:
`RepeatedField<T>::Assign` was added in [Protobuf v3.16.0](https://github.com/protocolbuffers/protobuf/releases/tag/v3.16.0).